packages/p2p/src/index.js

```diff
@@ -1 +1 @@
-export { default as P2P } from './components/app.jsx';
+export { default } from './components/app.jsx';
```

Comment:
i think it would be better to keep it as it is, since we do our exports like this usually and instead we use name import inside `packages/cashier/src/constants/routes-config.ts`.